lizrd/scripts/grid.py

```diff
@@ -37,34 +37,46 @@
         raise ValueError("No config path specified. Aborting...")
 
     if path.endswith(".yaml"):
```

Comment:
I do not understand. The path should point to a single .yaml file, yet you load multiple yamls?